src/components/datasets-list/datasets-item-info/index.tsx

```diff
@@ -1,10 +1,10 @@
 'use client';
 
+import { LuInfo } from 'react-icons/lu';
```

Comment:
Unify all libraries, it should be from heroicons. Remove `react-icons`

Reply:
I used heroicons and after agreed with Ari to use react-icons so I changed everything